reana-ui/src/pages/error/NotFound.js

```diff
@@ -22,7 +22,7 @@ export default function NotFound() {
           <h2>Page Not Found</h2>
           <p>The page you are looking for could not be found.</p>
           <Link to="/">
-            <Button primary>Go to Home</Button>
+            <Button primary>Go to Home page</Button>
```

Comment:
`s/Home/home/`  to use regular phrase capitalisation.

I would even use "Page not found" (BrE) rather than "Page Not Found" (AmE).

Reply:
I guess it's a matter of taste, for me the page looked better without these changes